### DISCUSSION TITLE
URGENT: Can I trust Git Credential Manager by GitCredentialManager

### DISCUSSION THREAD

**Discussion by jfcgno** [community/community#30885] [New to GitHub]
I've used Github desktop for a couple of years. Today it will suddenly would not log in to my account, although I have not changed password lately. When I click login with browser I'm taken to a page that says "Git Credential Manager by GitCredentialManager wants to access your account" and that I should authorize full read and write access.

Googling this has not given me any clues to why this would suddenly happen. Do I have to start using this service to login to Github desktop?

**Comment by wonhyeongseo** [comment]
Yes, pretty sure it's safe. Please check [this blog](https://github.blog/2022-04-07-git-credential-manager-authentication-for-everyone/) by the Github team.

**Reply by jfcgno** [reply]
Thanks for your answer wonhyeongseo. I do not find any mention of this being the new requirement for Github desktop in the blog post. In the documentation on how to login to Github desktop there is no mention of this credential manager - [https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/authenticating-to-github](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/authenticating-to-github) - I'm struggling to understand why this is happening without notification or information about the change and still unsure if it's trustworthy. If anyone has further info about this please let me know.

**Reply by wonhyeongseo** [reply]
@jfcgno If Github is trying canary deployment on some users, maybe restarting and [renewing your ip](https://computing.cs.cmu.edu/desktop/ip-renew#:~:text=The%20ipconfig%20command&text=Go%20to%20%22Start%20%3E%20Run%22,Enter%22%20to%20close%20the%20window) will help. You may have to do this a few times to get the original version that worked for you.

**Reply by jfcgno** [reply]
Thanks for the suggestion. This is happening on both my Windows machines so seems like it's not IP related maybe it's just my user :). It's not happening on my colleagues mac connecting to the same account btw.

**Reply by wonhyeongseo** [reply]
Github should really say something about this sooner or later. Maybe on the bright side, your windows machines may be one of the first canaries 😆 I think that's cool.

**Reply by jfcgno** [reply]
He he thanks :D
